conda/common/path.py

```diff
@@ -178,6 +178,8 @@ def get_python_site_packages_short_path(python_version):
         return 'lib/python%s/site-packages' % py_ver
 
 
+_VERSION_REGEX = re.compile(r"[0-9]*\.[0-9]*")
```

Comment:
I guess this should be one-or-more rather than zero-or-more?
```suggestion
_VERSION_REGEX = re.compile(r"[0-9]+\.[0-9]+")
```